### DISCUSSION TITLE
Project Based Notes

### DISCUSSION THREAD

**Discussion by raahimSal** [bramses/bramses-highly-opinionated-vault-2023#6] [Q&A] [answered]
When working on a project, where and how should project notes be stored?

**Comment by bramses** [comment, marked as answer]
Generally, I'd recommend taking advantage of the `QuickAdd: Create New Project` command to create a new project and taking advantage of the `projects` key in the YAML to automagically link the Resources to a project. For example:

In `Projects/{PROJ_NAME}/Resources.md`:

```
\`\`\`dataview
list 
where contains(projects, "Making-Mobile-YouTube-Publish-Better-Obsidian-CSS")
\`\`\`
```

In `Readwise/Articles/Make YouTube Video Embeds Responsive`: (note the projects YML key matching the one in Resources)

![Screenshot 2023-01-18 02-58-26](https://user-images.githubusercontent.com/3282661/213115600-0bef1e6e-deae-4be4-beaf-60b1cee64513.png)



## Result

![Screenshot 2023-01-18 02-58-10](https://user-images.githubusercontent.com/3282661/213115372-3ac1c160-e89e-47d2-bd20-9c16b019b5e3.png)

**Reply by raahimSal** [reply]
perfect okay thats what i was doing I was just wondering if there would be a specific folder corresponding to a given project and the yaml would be automagically filled out, but this works as well. Thanks
